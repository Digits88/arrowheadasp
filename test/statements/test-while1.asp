Test While statement

<pre>
Dim i               <% Dim i %>

While i < 10        <% While i < 10 %>
    i =             <%= i %>
    i = i + 1       <% i = i + 1 %>
Wend                <% Wend %>

</pre>
