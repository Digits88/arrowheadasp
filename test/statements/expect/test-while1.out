Test While statement

<pre>
Dim i               

While i < 10        
    i =             
    i = i + 1       
Wend                
    i =             1
    i = i + 1       
Wend                
    i =             2
    i = i + 1       
Wend                
    i =             3
    i = i + 1       
Wend                
    i =             4
    i = i + 1       
Wend                
    i =             5
    i = i + 1       
Wend                
    i =             6
    i = i + 1       
Wend                
    i =             7
    i = i + 1       
Wend                
    i =             8
    i = i + 1       
Wend                
    i =             9
    i = i + 1       
Wend                

</pre>
